docs/content/latest/quick-start/build-apps/go/ysql-pgx-ssl.md

```diff
@@ -0,0 +1,213 @@
+
+To install the package locally, run the following commands:
+
+```sh
+$ mkdir yb-pgx
+$ cd yb-pgx
+$ go mod init hello
+$ go get github.com/jackc/pgx/v4
+```
+
+## Create the sample Go application
+
+Create a file `ybsql_hello_world.go` and copy the contents below.
+
+{{< note title="Note">}}
+The constants defined under the `const` block in the code below have the default values.
+You may need to change the values of `host`, `user` and `password` as per your YugabyteDB cluster setup.
+For a **Yugabyte Cloud** cluster, the `host` will have a value in the format similar to `xxxx-xxxx-xxxx-xxxx-xxxx.aws.ybdb.io`.
+{{< /note >}}
+
+```go
+package main
+
+import (
+  "context"
+  "fmt"
+  "log"
+  "os"
+
+  "github.com/jackc/pgx/v4"
+)
+
+const (
+  host     = "127.0.0.1"
+  port     = 5433
+  user     = "yugabyte"
+  password = "yugabyte"
+  dbname   = "yugabyte"
+)
+
+func main() {
+    // SSL/TLS config is read from env variables PGSSLMODE and PGSSLROOTCERT, if provided.
+    url := fmt.Sprintf("postgres://%s:%s@%s:%d/%s",
+                       user, password, host, port, dbname)
+    conn, err := pgx.Connect(context.Background(), url)
+    if err != nil {
+        fmt.Fprintf(os.Stderr, "Unable to connect to database: %v\n", err)
+        os.Exit(1)
+    }
+    defer conn.Close(context.Background())
+
+    var dropStmt = `DROP TABLE IF EXISTS employee`;
+
+    _, err = conn.Exec(context.Background(), dropStmt)
+    if err != nil {
+        fmt.Fprintf(os.Stderr, "Exec for drop table failed: %v\n", err)
+    }
+
+    var createStmt = `CREATE TABLE employee (id int PRIMARY KEY,
+                                             name varchar,
+                                             age int,
+                                             language varchar)`;
+    _, err = conn.Exec(context.Background(), createStmt)
+    if err != nil {
+        fmt.Fprintf(os.Stderr, "Exec for create table failed: %v\n", err)
+    }
+    fmt.Println("Created table employee")
+
+    // Insert into the table.
+    var insertStmt string = "INSERT INTO employee(id, name, age, language)" +
+        " VALUES (1, 'John', 35, 'Go')";
+    _, err = conn.Exec(context.Background(), insertStmt)
+    if err != nil {
+        fmt.Fprintf(os.Stderr, "Exec for create table failed: %v\n", err)
+    }
+    fmt.Printf("Inserted data: %s\n", insertStmt)
+
+    // Read from the table.
+    var name string
+    var age int
+    var language string
+    rows, err := conn.Query(context.Background(), "SELECT name, age, language FROM employee WHERE id = 1")
+    if err != nil {
+        log.Fatal(err)
+    }
+    defer rows.Close()
+    fmt.Printf("Query for id=1 returned: ");
+    for rows.Next() {
+        err := rows.Scan(&name, &age, &language)
+        if err != nil {
+           log.Fatal(err)
+        }
+        fmt.Printf("Row[%s, %d, %s]\n", name, age, language)
+    }
+    err = rows.Err()
+    if err != nil {
+        log.Fatal(err)
+    }
+}
+```
+
+## Disable SSL/TLS
```

Comment:
The ysql-pq-ssl version is I think better - it doesn't bother with this 'Disable SSL/TSL section', and it doesn't appear to be needed here.

Reply:
same explanation as the previous one.